README.md

```diff
@@ -364,3 +364,38 @@ descriptions, try:
 ```
 $ python sample_indicator.py --help
 ```
+# Intergrating with OpenHIM (Middle ware component)
+For the Case ,where there are several OpenMRS systems ,
+OpenHIM can be used as a middle ware component to intergrate/track/log requests from several OpenMRS instaces.
+
+#### 1. Fire up OpenHIM Instance
```

Comment:
would be nice to add the screenshot of the architecture/dataflow that you showcased during the demo

Reply:
do you want to also add a link to the wikipage that contains more description?

Reply:
i think i can just provide a link to the demo video , the screenshots may make the read me too bulky. probably when we migrate the read me to the wiki , ill add screen shots